docs/develop/dapps/asset-processing/usdt.md

```diff
@@ -0,0 +1,42 @@
+# USDT Processing
+
+## Tether
+
+Stablecoins are a type of cryptocurrency whose value is pegged to another asset, such as a fiat currency or gold, to maintain a stable price. E.g. USD₮ stablecoin is issued by the company [Tether](https://tether.to/en/) in the form of ERC-20 tokens that move across the blockchain just as easily as other digital currencies but that are pegged to real-world currencies on a 1-to-1 basis (e.g., 1 USD₮ = 1 USD) and are backed 100% by Tether’s reserves. This offers traders, merchants and funds a low volatility solution when exiting positions in the market.
+
+In TON Blockchain USD₮ supported as [Jetton](/develop/dapps/asset-processing/jettons) jUSDT.
+
+:::info
+To integrate Tether’s USD₮ Token on TON Blockchain use the contract address:
+[EQCxE6mUtQJKFnGfaROTKOt1lZbDiiX1kCixRv7Nw2Id_sDs](https://tonviewer.com/EQCxE6mUtQJKFnGfaROTKOt1lZbDiiX1kCixRv7Nw2Id_sDs?section=jetton)
+:::
+
```

Comment:
Let's add more "Use" buttons for calls to action. Some will integrate (projects), while others will process (exchanges, mainly exchanges).

Here are 2 flows:

1. Leads to assets SDK:
https://github.com/ton-community/assets-sdk

2. Leads to processing:
https://docs.ton.org/develop/dapps/asset-processing/jettons

Example of buttons: https://github.com/ton-community/ton-docs/blob/main/docs/develop/smart-contracts/README.mdx?plain=1#L18